test/udp_sender.js

```diff
@@ -92,6 +93,31 @@ describe('udp sender should', () => {
         sender.flush();
     });
 
+    it ('span references serialize', (done) => {
+        let context = tracer.startSpan('just-used-for-context').context();
+        let context2 = tracer.startSpan('just-used-for-context').context();
+        let spanOne = tracer.startSpan('operation-one', {
+            references: [
+                new opentracing.Reference(opentracing.REFERENCE_CHILD_OF, context),
+                new opentracing.Reference(opentracing.REFERENCE_FOLLOWS_FROM, context2)
+            ]
+        });
+        spanOne.finish(); // finish to set span duration
+        spanOne = ThriftUtils.spanToThrift(spanOne);
+
+        server.on('message', (msg, remote) => {
+            let thriftObj = thrift.Agent.emitBatch.argumentsMessageRW.readFrom(msg, 0);
+            let batch = thriftObj.value.body.batch;
+            assert.isOk(batch);
+            assert.isOk(TestUtils.thriftSpanEqual(spanOne, batch.spans[0]));
```

Comment:
this is not actually testing that the refs were serialized, e.g. if there's a bug in startSpan that ignores them then the spans will still be equal